load-test/bin/complete-db-reset.sql

```diff
@@ -0,0 +1,40 @@
+
+/************************
+  resets common tables after a load-test
+  some statements should be excluded if you want to build up volume data
+ */
+
+TRUNCATE TABLE mtc_admin.adminLogonEvent
+TRUNCATE TABLE mtc_admin.psychometricianReportCache
+TRUNCATE TABLE mtc_admin.anomalyReportCache
+TRUNCATE TABLE mtc_admin.pupilRestart
+TRUNCATE TABLE mtc_admin.azureBlobFile
+TRUNCATE TABLE mtc_admin.auditLog
+-- user data
+TRUNCATE TABLE mtc_admin.answer
+TRUNCATE TABLE mtc_admin.checkPin
+TRUNCATE TABLE mtc_admin.checkConfig
+TRUNCATE TABLE mtc_admin.pupilAccessArrangements
+TRUNCATE TABLE mtc_admin.pupilAttendance
+DELETE FROM mtc_admin.pupilColourContrasts
+DELETE FROM mtc_admin.pupilResultsDiagnosticCache
+DELETE FROM mtc_admin.pupilFontSizes
```

Comment:
This is nothing to do with pupils! Actually, this is a lookup table that holds font size codes. Do not delete.